python/xorbits/_mars/dataframe/base/__init__.py

```diff
@@ -78,6 +79,7 @@ def _install():
         setattr(t, "drop", df_drop)
         setattr(t, "pop", df_pop)
         setattr(t, "pivot", df_pivot)
+        setattr(t, "pivot_table", df_pivot_table)
```

Comment:
`pivot_table` is also a module member of `python/xorbits/_mars/dataframe/__init__.py`

Reply:
done